core/templates/dev/head/pages/exploration_editor/post_publish_modal_directive.html

```diff
@@ -47,6 +61,11 @@ <h3>Awesome!</h3>
     <p>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
+    <div class="oppia-share-publish-link">
+      Here's your exploration link :
+     <br>
```

Comment:
The indentation is wrong. It should be indented by 2 from the outer scope.

Also, avoid using br for styling. Use CSS instead.

Reply:
Done.